src/app/cats/cats-detail-view/cats-detail-view.component.html

```diff
@@ -14,10 +21,12 @@
     <ng-container *ngIf="adoptionFormActive">
         <app-cats-adoption-form
                 [catsName]="cat.name"
-                (closeMe)="onFormClose($event)">
+                (click) = "reserve()"
```

Comment:
nie tutaj, samo kliknięcie na formularz to za mało, co jeśli ktoś go potem anulował?